examples/static_graph_optimizations/cifar/README.md

```diff
@@ -0,0 +1,26 @@
+# Convolutional neural networks for CIFAR-10 and CIFAR-100 Classification with static subgraph optimization
+
+This is an example of a convolutional neural network (convnet) applied to an image classification task using the CIFAR-10 or CIFAR-100 dataset. The CIFAR datasets can be a good choice for initial experiments with convnets because the size and number of images is small enough to allow typical models to be trained in a reasonable amount of time. However, the classification task is still challenging because natural images are used.
+
+Specifically, there are 50000 color training images of size 32x32 pixels with either 10 class labels (for CIFAR-10) or 100 class labels (for CIFAR-100).
+
+For CIFAR-10, state of the art methods without data augmentation can achieve similar to human-level classification accuracy of around 94%.
+For CIFAR-100, state of the art without data augmentation is around 20% (DenseNet).
+
+The code consists of three parts: dataset preparation, network and optimizer definition and learning loop, similar to the MNIST example.
+
+More models may be added in the future, but currently only one model is available that uses the VGG-style network from [here](http://torch.ch/blog/2015/07/30/cifar.html) which is based on the network architecture from the paper from [here](https://arxiv.org/pdf/1409.1556v6.pdf).
```

Comment:
This uses the VGG-style...

Reply:
sorry, I edited the corresponding readme of the CIFAR example (not the one in static graph example) by mistake. I'll updated this one, too.